alpheios_nemo_ui/data/assets/package.json

```diff
@@ -14,6 +14,8 @@
     "bootstrap": "^4.1.3"
   },
   "devDependencies": {
+    "alpheios-components": "github:alpheios-project/components",
```

Comment:
I'm not sure we need this because embed-lib has its own dependency on components and components will get installed through it anyway

alpheiso-components is a dev dependency but it will still gets installed with `npm install`. But I'm wondering if it's worth moving components into "regular" dependencies inside the embed-lib package because we're linking it dynamically now and it is required for the embed-lib execution. What do you think?

Reply:
yes, you're probably right here.